services/src/contexts/session.rs

```diff
@@ -33,6 +33,14 @@ pub struct SimpleSession {
     id: SessionId,
     pub project: Option<ProjectId>,
     pub view: Option<STRectangle>,
+    #[serde(skip)]
+    is_admin: bool,
```

Comment:
You could always convert an AdminSession to a session. This is not new. Only now you know that it is an admin authenticated.

Reply:
Okay, but why is there an `AdminSession` at all? It could be just the SimpleSession with that flag.
We shouldn't have both I guess.

Reply:
We use the `AdminSession` to implement methods that can only be called by admins.

Reply:
Maybe it better matches with the other Session if we omit the AdminSession and just check for this flag?
It somehow feels strange to have both types if one has all the necessary information.

Reply:
Then we would expect regular sessions in the admin handlers, and would have to check for the is_admin flag in every one. This would be more code in every single admin handler, because we no longer check the admin status in the type.

Also: currently the UserSession does not have the is_admin flag, because we don't need it for deleting datasets because we have permissions there to decide whether a user is allowed to delete a dataset.

Reply:
But for the UserSession we actually don't need "admins" as a separate type. We could have just gotten the right to do it, don't we?

Reply:
Add TODO e.g. that this will be most likely removed in the future

Reply:
0ca45624d8989b8c1a26f06879e31669e48e25f5